components/theia/packages/gitpod-extension/src/browser/extensions/gitpod-plugin-server.ts

```diff
@@ -63,19 +67,18 @@ export class GitpodPluginServer implements PluginServer {
     protected async setStorage(storage: KeysToKeysToAnyValue, kind: PluginStorageKind): Promise<boolean> {
         const uri = await this.getUri(kind);
         this.localCache.set(uri, storage);
-        const service = await this.serviceProvider.getService();
-        service.server.updateUserStorageResource({ uri, content: JSON.stringify(storage) }).catch(e => console.error(e));
+        await this.cachedUserStorage.write({ uri, content: JSON.stringify(storage) });
```

Comment:
we should not await here

Reply:
I added because:
 1. it is no long call behind that
 1. no errors
 1. I thought the original no-`await` was made because it feared long-term roundtrips + potential errors bubbling up.

If we want to make clear that's still meant to explicitly be separate we can still replace with the `.ctach(console.error)`, though.